app/views/users/prompt_urs_association.html.erb

```diff
@@ -3,8 +3,8 @@
     <div class="content">
       <section>
         <fieldset>
-          <h2>Redirecting to URS</h2>
-          <p>It appears you do not have a URS account linked with your Launchpad account. MMT and CMR permissions (ACLs) are based on your URS account. Please click the button below to log in to the URS account you listed on your NAMS request and would like to associate with your Launchpad account.</p>
+          <h2>Redirecting to Earthdata Login</h2>
+          <p>It appears you do not have an  Earthdata Login account linked with your Launchpad account. MMT and CMR permissions (ACLs) are based on your Earthdata Login account. Please click the button below to log in to the Earthdata Login account you listed on your NAMS request and would like to associate with your Launchpad account.</p>
```

Comment:
It looks like there is an extra space before the first 'Earthdata Login' which is causing the test to fail

Reply:
Also you're using an 'an' here and the test is looking for 'a' before 'Earthdata Login'.